connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/UserConnectorImpl.java

```diff
@@ -42,6 +48,28 @@ public List<InstitutionInfo> getUserProducts(String userId) {
         return result;
     }
 
+    @Override
+    public User getUserById(String userId) {
+        log.trace("getUserById start");
+        log.debug("getUserById id = {}", userId);
+        User user = userMapper.toUser(userApiRestClient._usersIdDetailsGet(userId).getBody());
+        log.debug(LogUtils.CONFIDENTIAL_MARKER, "getUserById = {}", user);
+        log.trace("getUserById end");
+        return user;
+    }
+
+    @Override
+    public User searchByFiscalCode(String fiscalCode) {
+        log.trace("searchByFiscalCode start");
+        log.debug(LogUtils.CONFIDENTIAL_MARKER, "searchByFiscalCode fiscalCode = {}", fiscalCode);
+        SearchUserDto request = new SearchUserDto();
```

Comment:
We could use @Builder and avoid to declare object in a row and set a field in a second one